### ISSUE TITLE
1.12 - Avoid Detection - How To -  Automate Logging On/Off

### ISSUE THREAD

**Issue by rhyys5** [UltimateFishbot/UltimateFishbot#154]
Hello all,

I was using the bot, works great!  However, I was watching movies while fishing when a player appeared next to me (obviously a GM) and teleported me across the lake.  I started moving around and the player left.  If I was away from the computer at that time I probably would have been banned.  So, I decided to create a batch file to accomplish these goals.

- Log into the game
- Start Fishing
- Fish for 10 minutes (Modifiable)
- Log off of the game

Below is the .bat and .vbs file:  

**NOTES:** 

You will need to modify drives, paths, login, and START/STOP fishing key info in the 2 files

The "TIMEOUT /T 600" in the .bat file determines how long you fish before it logs you off

Just schedule the .bat file to run as often as you like.  Example, if you set the .bat file to run every 45 minutes, you will fish for 10 minutes, and be logged off for 35 minutes until you start fishing again.  This should make you look less suspicious. Hope you find this helpful!!!

The TIMEOUT /T 600 in the .bat file determines how long you fish before it logs you off

**fishing.bat**

Start "" "*PATH*\wow.exe"
TIMEOUT /T 3
*PATH*
TIMEOUT /T 1
*DRIVE*
TIMEOUT /T 1
cd *PATH*
TIMEOUT /T 1
Start "" "Ultimatefishbot.exe"
TIMEOUT /T 2
Start "" "*PATH*\fishing.vbs"
TIMEOUT /T 600
taskkill /im "Ultimatefishbot.exe" /f
TIMEOUT /T 1
taskkill /im "WoW.exe" /f
TIMEOUT /T 1
taskkill /im "cmd.exe" /f

**fishing.vbs**

WScript.Sleep 2000
Set objShell = CreateObject("WScript.Shell")
objShell.AppActivate "World of Warcraft"
WScript.Sleep 1000
Set WshShell = WScript.CreateObject("WScript.Shell")
WshShell.SendKeys "USER NAME"
WScript.Sleep 1000
WshShell.SendKeys "{TAB}"
WScript.Sleep 1000
WshShell.SendKeys "PASSWORD"
WScript.Sleep 1000
WshShell.SendKeys "{ENTER}"
WScript.Sleep 3000
WshShell.SendKeys "{ENTER}"
WScript.Sleep 9000
Set objShell = CreateObject("WScript.Shell")
objShell.AppActivate "World of Warcraft"
Set WshShell = WScript.CreateObject("WScript.Shell")
WshShell.SendKeys "{*START/STOP Key*}"

**Comment by NaNraptor** [comment]
Thank you for this, could you create a pull request adding those two files?

I will merge it with the main repo when you do so.

**Comment by LaBlazer** [comment]
I don't think this should be included in the main repo. If this functionality is needed it should be added to the bot in C# and not in 2 different scripts in different languages...

**Comment by NaNraptor** [comment]
Agreed, but before someone decides to include it as a feature (which will probably take a very long time) I think it won't be such a bad idea to add it, then remove it later.

Or we can keep it here. I will leave this one to you @LaBlazer ;)
